.changes/v3.7.0/851-features.md

```diff
@@ -0,0 +1 @@
+* Add `catalog_item_metadata` attribute to resource and data source `catalog_item` to support metadata on CatalogItem entities [GH-851]
```

Comment:
let's keep consistent in changelog format where mentioned resource or data and what changed in that, for e.g.: 
* `vcd_nsxt_network_dhcp` resource and data source now supports `dns_servers` [GH-830]

Reply:
+1, it actually helps when they are at the beginning of the sentence also during release, i.e. when preparing the announcements.

Reply:
Done, thanks for noticing!